test/instances/reth/RETHHyperdrive.t.sol

```diff
@@ -28,224 +28,78 @@ import { RETHTarget3Deployer } from "contracts/src/deployers/reth/RETHTarget3Dep
 import { RETHTarget4Deployer } from "contracts/src/deployers/reth/RETHTarget4Deployer.sol";
 import { stdStorage, StdStorage } from "forge-std/Test.sol";
 
-contract RETHHyperdriveTest is HyperdriveTest {
+contract RETHHyperdriveTest is IntegrationTest {
     using FixedPointMath for uint256;
     using Lib for *;
     using stdStorage for StdStorage;
 
-    uint256 internal constant FIXED_RATE = 0.05e18;
+    // Rocket Network contracts can be upgraded and addresses changed.
+    // We can safely assume these addresses are accurate because
+    // this testing suite is forked from block 19429100.
     IRocketStorage internal constant ROCKET_STORAGE =
         IRocketStorage(0x1d8f8f00cfa6758d7bE78336684788Fb0ee0Fa46);
-
-    IRocketTokenRETH rocketTokenRETH;
-    IRocketNetworkBalances rocketNetworkBalances;
-    IRocketDepositPool rocketDepositPool;
-
+    IRocketTokenRETH internal constant rocketTokenRETH =
+        IRocketTokenRETH(0xae78736Cd615f374D3085123A210448E74Fc6393);
+    IRocketNetworkBalances internal constant rocketNetworkBalances =
+        IRocketNetworkBalances(0x07FCaBCbe4ff0d80c2b1eb42855C0131b6cba2F4);
+    IRocketDepositPool internal constant rocketDepositPool =
+        IRocketDepositPool(0xDD3f50F8A6CafbE9b31a427582963f465E745AF8);
+
+    // Whale accounts.
     address internal RETH_WHALE = 0xCc9EE9483f662091a1de4795249E24aC0aC2630f;
-    address internal ETH_WHALE = 0x00000000219ab540356cBB839Cbe05303d7705Fa;
+    address[] internal whaleAccounts = [RETH_WHALE];
+
+    // The configuration for the integration testing suite.
+    IntegrationConfig internal __testConfig =
+        IntegrationConfig(
+            whaleAccounts,
+            IERC20(rocketTokenRETH),
+            IERC20(ETH),
+            0,
+            1e16,
+            POSITION_DURATION,
+            false
+        );
 
-    HyperdriveFactory factory;
-    address deployerCoordinator;
+    constructor() IntegrationTest(__testConfig) {}
 
     function setUp() public override __mainnet_fork(19_429_100) {
+        // Give the rETH contract ETH to mimic adequate withdrawable liquidity.
```

Comment:
This comment doesn't describe the following `super.setUp();`. We should either update this comment to describe both operations or split them out.